shared/karma/shared.karma.conf.js

```diff
@@ -94,10 +94,8 @@ function getConfig(config, env) {
   if (launchers) {
     overrides.customLaunchers = launchers;
     overrides.browsers = Object.keys(launchers);
-    overrides.browserStack = {
-      username: env.BROWSER_STACK_USERNAME,
-      accessKey: env.BROWSER_STACK_ACCESS_KEY
-    };
+    overrides.browserStack.username = env.BROWSER_STACK_USERNAME;
```

Comment:
We were incorrectly overriding the entire `browserStack` object, so the increased timeout was being wiped out.